hassio/src/supervisor-base-element.ts

```diff
@@ -85,20 +86,20 @@ export class SupervisorBaseElement extends urlSyncMixin(
     );
 
     if (atLeastVersion(this.hass.config.version, 2021, 2, 4)) {
-      Object.keys(supervisorStore).forEach((store) => {
+      Object.keys(supervisorCollection).forEach((store) => {
```

Comment:
We should take the renaming a bit further, also rename the events, etc. to avoid confusion.
```suggestion
      Object.keys(supervisorCollection).forEach(colllection) => {
```